rbc/tests/heavydb/test_nrt_set.py

```diff
@@ -0,0 +1,103 @@
+from rbc.tests import heavydb_fixture
+import pytest
+
+rbc_heavydb = pytest.importorskip('rbc.heavydb')
+available_version, reason = rbc_heavydb.is_available()
+pytestmark = pytest.mark.skipif(not available_version, reason=reason)
+
+
+@pytest.fixture(scope='module')
+def heavydb():
+    for o in heavydb_fixture(globals(), load_test_data=False):
+        define(o)
+        yield o
+
+
+def define(heavydb):
+    # Defines simple methods just to verify if NRT is working and if any there
+    # are any symbol
+    #
+    # List of methods:
+    # [X] set.add
+    # [X] set.clear
+    # [X] set.copy
+    # [X] set.difference
+    # [X] set.difference_update
+    # [X] set.discard
+    # [X] set.intersection
+    # [X] set.intersection_update
+    # [X] set.isdisjoint
+    # [X] set.issubset
+    # [X] set.issuperset
+    # [X] set.pop
+    # [X] set.remove
+    # [X] set.symmetric_difference
+    # [X] set.symmetric_difference_update
+    # [ ] set.union
```

Comment:
What blocks supporting `set.union`?

Reply:
It doesn't seem to work in Numba:
```python
self = <numba.core.typeinfer.TypeInferer object at 0x7ff2e5965f30>, raise_errors = True

    def propagate(self, raise_errors=True):
        newtoken = self.get_state_token()
        oldtoken = None
        # Since the number of types are finite, the typesets will eventually
        # stop growing.

        while newtoken != oldtoken:
            self.debug.propagate_started()
            oldtoken = newtoken
            # Errors can appear when the type set is incomplete; only
            # raise them when there is no progress anymore.
            errors = self.constraints.propagate(self)
            newtoken = self.get_state_token()
            self.debug.propagate_finished()
        if errors:
            if raise_errors:
                force_lit_args = [e for e in errors
                                  if isinstance(e, ForceLiteralArg)]
                if not force_lit_args:
>                   raise errors[0]
E                   numba.core.errors.TypingError: Failed in nopython mode pipeline (step: nopython frontend)
E                   Invalid use of BoundFunction(set.union for set(int64)) with parameters (set(Literal[int](0)))
E
E                   During: resolving callee type: BoundFunction(set.union for set(int64))
E                   During: typing of call at /home/guilhermeleobas/git/rbc/rbc/tests/heavydb/test_nrt_set.py (84)
E
E
E                   File "rbc/tests/heavydb/test_nrt_set.py", line 84:
E                       def test_set(t, method):
E                           <source elided>
E                           elif method_str == 'union':
E                               return len(s.union({0})) == 6
E                               ^

../../miniconda3/envs/rbc/lib/python3.10/site-packages/numba/core/typeinfer.py:1086: TypingError
```